src/ray/protobuf/common.proto

```diff
@@ -288,7 +288,7 @@ message TaskArg {
   // Metadata for pass-by-value arguments.
   bytes metadata = 3;
   // ObjectIDs that were nested in the inlined arguments of the data field.
-  repeated bytes nested_inlined_ids = 4;
+  repeated ObjectReference nested_inlined_refs = 4;
```

Comment:
nits: consider use a new index to avoid any unforeseen compatibility issue

Reply:
Would that mean keeping the old `nested_inlined_ids` field around? I think I'd rather not do that since we don't currently support core workers on different versions anyway.

Reply:
I think this proto wouldn't have backward compat issue, so it should be fine?  (not an autoscaling path)